quickstack/manifests/compute_common.pp

```diff
@@ -284,4 +287,36 @@
            nova_uuid     => $nova_uuid,
            rbd_key       => $rbd_key,
         }
+#Custimization for ruby. Ruby is necessary dependency of sensu
```

Comment:
@ravisantoshgudimetla I failed to install one necessary gem for running sensu plugin before with error:

 "ERROR: Could not find a valid gem 'sensu-plugin' (>= 0), here is why: Unable to download data from https://rubygems.org/ ".

Then I tried updating the packages list and  it works afterward. So I think I have to make sure the rubygem would have the latest package list, that could be achieve by running "gem update --system".